also include $LD_LIBRARY_PATH in environment where "python -m pip freeze" is run on 'import qiskit'
author: Kenneth Hoste (HPC-UGent)
--- qiskit/version.py.orig	2021-08-04 16:34:17.261482728 +0200
+++ qiskit/version.py	2021-08-04 16:34:26.087460624 +0200
@@ -25,7 +25,7 @@
 def _minimal_ext_cmd(cmd):
     # construct minimal environment
     env = {}
-    for k in ["SYSTEMROOT", "PATH"]:
+    for k in ["SYSTEMROOT", "PATH", "LD_LIBRARY_PATH"]:
         v = os.environ.get(k)
         if v is not None:
             env[k] = v
